db/seeds.rb

```diff
@@ -5,3 +5,16 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+Actor.create(name: 'Aisha Khan')
```

Comment:
Is there a better way of writing this? what happens if you had like 100 or 1000 entries to seed.

Reply:
By using csv files and then reading data from them?